bedrock/press/templates/press/press-inquiry.html

```diff
@@ -0,0 +1,103 @@
+{% extends "base-resp.html" %}
+
+{% block page_title %}{{_('Press Inquiry Form')}}{% endblock %}
+{% block body_id %}press_inquiry{% endblock %}
+{% block body_class %}sand{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'press' %}
+  <!--[if lte IE 9]>
+    {# Only needed for IE before v10 #}
+    {% stylesheet 'press-ie9' %}
+  <![endif]-->
+{% endblock %}
+
+{% block content %}
+
+<main role="main" id="main-content">
+  <header class="main-header">
+    <h1 class="title-shadow-box">{{_('Press Inquiry')}}</h1>
+  </header>
+
+  <section class="container">
+
+    {% if not form_success %}
+      <h2 class="secondary-title">{{ _('Please complete and submit the form below with your press inquiry.') }}</h2>
+    {% endif %}
+
+    <div class="main-column">
+    {% if not form_success %}
+
+      {% if form.errors %}
+        <div class="tooltip">
+          {{ _('An error has occurred with your submission. Please review your information below and try again.') }}
+        </div>
+      {% endif %}
+
+      <form name="mozilla-contact-us" id="press_inquiry" class="press-inquiry" action="{{ url('press.press-inquiry') }}" method="post">
```

Comment:
Seeing I do not actually use it for anything, I am just removing it.  Nice catch